spec/spec_helper.rb

```diff
@@ -43,13 +43,13 @@
 ::IronTrailSpecMigrator.new.migrate
 
 Time.now.tap do |date|
-  partition_name = "irontrail_chgn_#{date.strftime('%Y%m')}"
+  partition_name = "irontrail_chgn_infinite"
```

Comment:
the goal isn't to test `pg_party` itself or partitioning. So since I'm now also using dates far in the past in some tests, I'll just create one big partition to accomodate them all.